google/cloud/forseti/common/gcp_type/groups_settings.py

```diff
@@ -79,5 +79,6 @@ def from_json(cls, email, settings):
             whoCanViewMembership=settings.get('whoCanViewMembership'),
             whoCanViewGroup=settings.get('whoCanViewGroup'),
             whoCanInvite=settings.get('whoCanInvite'),
-            allowExternalMembers=settings.get('allowExternalMembers'),
+            allowExternalMembers=str(
+                settings.get('allowExternalMembers', '')).lower() == 'true',
```

Comment:
Is there a reason this is converted to a string and compared to a string vs. left as bool and compared to bool?

Reply:
`settings.get('allowExternalMembers', '')` is the response from the GSuite API, which returns this value as a string instead of a bool. For example `allowExternalMembers="true"`. I am converting the string to a string to future proof, in case of one day this is correctly returned as a bool.

Reply:
Ok thanks for the explanation!